src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -12,8 +16,23 @@ export default function FacilityAddress({
   showPhone = true,
   level = 4,
 }) {
+  const { id } = useParams();
+  const appointment = useSelector(
+    state => selectAppointmentById(state, id),
+    shallowEqual,
+  );
+  const featureCovid19Vaccine = useSelector(state =>
+    selectFeatureCovid19Vaccine(state),
+  );
+  const covidPhone = facility?.detailedServices
+    ? facility?.detailedServices[0]?.appointmentPhones[0]?.number
```

Comment:
I'm not sure we want to rely on covid always being the first item in the detailed services array. It might be better to make a helper to find the detailedService object by `name` and return the phone number. Or do that in the transformer and set the phone number on the facility object somewhere.